doc/includes/reveal/examples_reveal_modal_examples.html

```diff
@@ -25,7 +25,7 @@ <h2 id="videoModalTitle">This modal has video</h2>
     <iframe width="1280" height="720" src="//www.youtube-nocookie.com/embed/wnXCopXXblE?rel=0" frameborder="0" allowfullscreen></iframe>
   </div>
 
-  <a class="close-reveal-modal" aria-label="Close">&#215;</a>
+  <a class="close-reveal-modal" aria-label="Close" role="button" tabindex="0">&#215;</a>
```

Comment:
This should be a tag. aria-haspopup and role="button" not needed. tabindex="0" not needed because anchor tags are already in the tab order.